apps/samples/Ensemble8/src/samples/java/ensemble/samples/graphics2d/puzzle/Piece.java

```diff
@@ -43,9 +42,10 @@
 import javafx.scene.shape.Ellipse;
 import javafx.scene.shape.Rectangle;
 import javafx.scene.shape.Shape;
-   /**
-     * Node that represents a puzzle piece
-     */
+
+/**
+ * Node that represents a puzzle piece
+ */
```

Comment:
Hmm. This is more than just fixing imports. Were these lines reformatted because they were touching the modified import lines? Anything other than changes to the import statements will require extra time / mental energy to review.

Reply:
this is just one spot that bothered me.  reformatted since I have to delete a blank line.

won't happen again, I promise.